ChangeLog.d/systematically_store_bit_size_3740.txt

```diff
@@ -0,0 +1,7 @@
+Changes
+   * PSA persistent storage format is updated to always store the key bits
+     attribute. No automatic upgrade path is provided. Previously stored keys
+     must be erased, or manually upgraded based on the key storage format
+     specification (mbed-crypto-storage-specification.md). The storage format
+     for the dynamic Secure Element driver has also been updated. Previously
+     stored keys must be erased, or manually upgraded. #3740
```

Comment:
The part about secure elements is redundant.
```suggestion
   * The PSA persistent storage format is updated to always store the key bits
     attribute. No automatic upgrade path is provided. Previously stored keys
     must be erased, or manually upgraded based on the key storage format
     specification (docs/architecture/mbed-crypto-storage-specification.md). #3740
```

It's very likely that we'll also upgrade the format version to 1 before the next release anyway.

Reply:
Ok, thanks for the suggestion.